editors/code/package-lock.json

```diff
@@ -878,7 +878,7 @@
         "vscode-languageclient": {
             "version": "6.1.0",
             "resolved": "https://registry.npmjs.org/vscode-languageclient/-/vscode-languageclient-6.1.0.tgz",
-            "integrity": "sha512-Tcp0VoOaa0YzxL4nEfK9tsmcy76Eo8jNLvFQZwh2c8oMm02luL8uGYPLQNAiZ3XGgegfcwiQFZMqbW7DNV0vxA==",
+            "integrity": "sha1-7mfAt4GMQs4CgVctBcia38xPWjg=",
```

Comment:
Hm, I think this is a step in the wrong direction. Perhaps you need to update your node version?

Reply:
I also noticed this change, didn't understand why actually that happenned, I am not sure this is my node version, I use the latest node `13.7.0`

Reply:
Try removing your `node_modules`: https://github.com/rust-analyzer/rust-analyzer/pull/2961.

Reply:
Okay, thank you for the pointer!